translator.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * A super simple language translation tool
+ */
+
+// const translationDataExample = {
+//   'hello' : {
+//     'eng': 'Hello!',
+//     'spa': 'Hola!',
+//     'kar': '',
+//     'som': 'Salaan!',
+//     'hmn': 'Nyob zoo!'
+//   }
+// }
+
+class Translator {
+  attr = 'data-translation-id'
+
+  constructor(translations) {
+    this.translations = translations
+  }
+
+  // this converts a `gsx$keyname` to `keyname
+  convertGoogleKey(key) {
```

Comment:
Is this just stubbed out or is it overridden somewhere? It's not clear to me why this is returning null or undefined.